cmd/lncli/cmd_invoice.go

```diff
@@ -234,6 +234,18 @@ var listInvoicesCommand = cli.Command{
 			Usage: "if set, invoices succeeding the " +
 				"index_offset will be returned",
 		},
+		cli.Uint64Flag{
+			Name: "creation_date_start",
+			Usage: "timestamp in seconds, if set, filter " +
```

Comment:
I think here is right because it says filter and in the other comment says "filter out".

Reply:
hmm yeah changed the `filter out` to be more clear.